packages/adapter-react/README.md

```diff
@@ -53,17 +53,38 @@ The generator will create a custom React component and context inside `i18n-reac
 Wrap your application root with the `TypesafeI18n` component:
 
 ```jsx
-import React from 'react'
+import React, { useState, useEffect } from 'react'
+import { detectLocale, navigatorDetector } from 'typesafe-i18n/detectors';
 import TypesafeI18n from './i18n/i18n-react'
+import { baseLocale, locales } from 'i18n/i18n-util';
+import { loadLocale } from './i18n/i18n-util.sync';
 
-function App() {
-
-   // TODO: load locales (https://github.com/ivanhofer/typesafe-i18n/tree/main/packages/generator#loading-locales)
 
+function App() {
+   // Detect locale
+   // (Use as advanaced locale detection strategy as you like. 
+   // More info: https://github.com/ivanhofer/typesafe-i18n/tree/main/packages/detectors)
+   const locale = detectLocale(
```

Comment:
You can use the `detectLocale` function from `i18n/i18n-util.ts`. Then you don't need to pass `baseLocale` and `locales`